zipkin-autoconfigure/ui/src/main/java/zipkin/autoconfigure/ui/ZipkinUiProperties.java

```diff
@@ -24,6 +24,8 @@
   private int defaultLookback = (int) TimeUnit.DAYS.toMillis(7);
   private String instrumented = ".*";
   private String logsUrl = null;
+  private float dependencyLowErrorRate = 0.5f; // 50% of calls in error turns line yellow
```

Comment:
I like those too. Just one thing: it is nice that rate (hopefully) conveys that the value should be between 0 and 1. Threshold could be (mis)taken to mean an absolute number of errors, perhaps.